modules/platforms/python/cpp_module/type_conversion.h

```diff
@@ -122,40 +161,162 @@ static void submit_pyobject(ignite::binary_tuple_builder &builder, PyObject *obj
         return;
     }
 
-    if (PyBool_Check(obj)) {
-        bool val = (obj == Py_True);
+    if (PyObject_IsInstance(obj, py_get_module_timestamp_class())) {
+        auto double_value = PyFloat_AsDouble(obj);
+        if (PyErr_Occurred()) {
+            throw ignite::ignite_error("Can not get ticks count from a TIMESTAMP value");
```

Comment:
Is that possible in practice? Should we add the original error message from `PyErr_Occurred`?

Reply:
It's not, I'll handle it just as a float type. Added getting an error message from an exception where possible.